pkgs/development/compilers/cudatoolkit/common.nix

```diff
@@ -252,13 +252,21 @@ backendStdenv.mkDerivation rec {
       mv pkg/builds/nsight_systems/target-linux-x64 $out/target-linux-x64
       mv pkg/builds/nsight_systems/host-linux-x64 $out/host-linux-x64
       rm $out/host-linux-x64/libstdc++.so*
+
     ''}
       ${lib.optionalString (lib.versionAtLeast version "11.8")
```

Comment:
I've just realised this should only be done for 12 onwards, since we don't replace Qt libs for 11.x, fixing.

Reply:
I think that should be preferred, unless it would be much more effort than what is done for this currently (a more general feature of dropping vendor dependencies vs. a bug-fix for `nsys-ui`). Thoughts, @graham33?